bool switch_on
string device_name
---
bool feedback
